src/Commands/field/FieldEntityReferenceHooks.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Drush\Commands\field;
+
+use Drupal\Core\Entity\EntityTypeBundleInfoInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Commands\DrushCommands;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class FieldEntityReferenceHooks extends DrushCommands
```

Comment:
```suggestion
final class FieldEntityReferenceHooks extends DrushCommands
```
Lets add final unless we expect folks to extend this.

Reply:
We do expect people to extend this, that's the whole point of this PR. This was explained in https://github.com/drush-ops/drush/pull/5556.